app/views/users/show.html.erb

```diff
@@ -0,0 +1,193 @@
+<div class="container">
+<% if logged_in? %>
```

Comment:
Right now you're hiding the whole page if a user isn't logged in, yeah? If that's what you intend, not logged in user should really just be redirected to the login page.